### DISCUSSION TITLE
Latency unit for explicit buckets

### DISCUSSION THREAD

**Discussion by BlueSeaOcean** [cloudprober/cloudprober#745] [Q&A]
Given following configs, does it mean the latency buckets (le) will be 0.5s, 1.0s, 5s, and 10s according?

```
probe{
 ...
 lentency_distribution {
  explicit_buckets: "0.5,1.0,5,10"
 }
 ...
}
...
latencyUnit = time.Second
```

**Comment by manugarg** [comment]
That's correct. Latency unit is define like this though:
```
latency_unit: "s" # seconds
```

The way it works is, we change latency into seconds (or whatever latency_unit is) and then put it in unit-less buckets.
